packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -44,25 +146,38 @@ const BaseButton = ({
   type = 'button',
   children,
 }: BaseButtonProps): ReactElement => {
-  const { theme } = useTheme();
+  const { theme, platform } = useTheme();
+  const buttonTypographyTokens = typography[platform];
```

Comment:
Seeing a eslint `no-unsafe-member-access` warning here - `Computed name [platform] resolves to an any value.`